packages/rs-drive-abci/src/execution/validation/state_transition/common/asset_lock/proof/validate/instant/mod.rs

```diff
@@ -45,12 +44,29 @@ impl AssetLockProofValidation for InstantAssetLockProof {
         // As we know this outpoint was already considered final and locked.
 
         if validation_mode != ValidationMode::RecheckTx && !validation_result.has_data() {
-            let is_instant_lock_signature_valid = self.instant_lock().verify_signature(
-                platform_ref.core_rpc,
-                platform_ref.state.last_committed_core_height(),
-            )?;
+            // We should be able to disable instant lock versification for integration tests
+            if platform_ref
+                .config
+                .testing_configs
+                .disable_instant_lock_signature_verification
+            {
+                return Ok(validation_result);
+            }
+
+            // TODO: Shouldn't we add an operation for fees?
```

Comment:
Shouldn't we add an operation to calculate fees?

Reply:
Actually, there are more places in asset lock verification is not covered with fees. I will do it as a separate PR.